client/src/components/kvEditor/KVEditorRow.tsx

```diff
@@ -157,17 +158,28 @@ function KVEditorRow({
 
   const eventHandlers = EditorView.domEventHandlers({
     paste(event) {
-      event.preventDefault();
-      if (!event.clipboardData) {
+      if (!event.target || !event.clipboardData) {
+        return;
+      }
+      // differenciates between initial and synthetic event to prevent infinite loop
+      if (!event.isTrusted) {
```

Comment:
how is `isTrusted` added to the event?

Reply:
`isTrusted` is true for native events and false for synthetic, code-generated events.
 
https://developer.mozilla.org/en-US/docs/Web/API/Event/isTrusted